sdk/log/provider_test.go

```diff
@@ -249,24 +249,22 @@ func TestWithResource(t *testing.T) {
 }
 
 func TestLoggerProviderConcurrentSafe(*testing.T) {
```

Comment:
@pellared @MrAlias let me know if you want this and the other SDK test changes reverted